ckanext/switzerland/helpers/plugin_utils.py

```diff
@@ -1,24 +1,22 @@
 """
 helpers of the plugins.py
 """
-import isodate
 import json
-import re
 import logging
-from ckan import logic
+import re
```

Comment:
What is re?

Reply:
It's the regular expressions library: https://docs.python.org/3/library/re.html